test/runtests.jl

```diff
@@ -113,21 +113,6 @@ end
     end
 end
 
-@testset "IsDeterministic with Vector" begin
-    ham = HubbardReal1D(BoseFS((1, 1, 1, 1)))
-    dim = dimension(ham)
-    sm, basis = Rimu.Hamiltonians.build_sparse_matrix_from_LO(ham, starting_address(ham))
-    @test dim == length(basis)
-    # run lomc! in deterministic mode with Matrix and Vector
-    a = lomc!(sm, ones(dim); threading=true).df # no actual threading is done, though
-    b = lomc!(sm, ones(dim); threading=false).df
-    @test a.shift ≈ b.shift
-    # run lomc! in deterministic mode with Hamiltonian and DVec
-    v = DVec(k=>1.0 for k in basis) # corresponds to `ones(dim)`
-    c = lomc!(ham, v).df
-    @test a.shift ≈ c.shift
-end
```

Comment:
I'm not sure we  need to keep this functional. However, is there a good reason to remove it?

Reply:
A very similar thing is tested in `test/Interfaces.jl`.